tools/tertiary-analysis/scanpy/scanpy-run-umap.xml

```diff
@@ -65,7 +65,7 @@ PYTHONIOENCODING=utf-8 scanpy-run-umap
 
   <outputs>
     <expand macro="output_data_obj" description="UMAP object"/>
-    <data name="output_embed" format="csv" from_work_dir="embeddings.csv" label="${tool.name} on ${on_string}: UMAP embeddings">
+    <data name="output_embed" format="tsv" from_work_dir="embeddings.tsv" label="${tool.name} on ${on_string}: UMAP embeddings">
```

Comment:
Here it might be better to use tabular as format instead of tsv.
```suggestion
    <data name="output_embed" format="tabular" from_work_dir="embeddings.tsv" label="${tool.name} on ${on_string}: UMAP embeddings">
```